tests/lib/rules/no-unnecessary-act.test.ts

```diff
@@ -0,0 +1,185 @@
+import { createRuleTester } from '../test-utils';
+import rule, { RULE_NAME } from '../../../lib/rules/no-unnecessary-act';
+
+const ruleTester = createRuleTester({
+  ecmaFeatures: {
+    jsx: true
```

Comment:
Our `createRuleTester` already enables `jsx` so you don't need to set it here :)